ansible/vagrant/README.md

```diff
@@ -199,6 +199,19 @@ kube-master | SUCCESS => {
 ...
 ```
 
+Example to launch one playbooks:
```

Comment:
Can you mentioned that information with the example? Plus, I don't think it will work unless you specify vagrant's inventory file. Which is at ``.vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory``.